parseabc.h

```diff
@@ -74,7 +74,8 @@ extern void print_inputline_nolinefeed();
 extern void parseron();
 extern void parseroff();
 
-extern int lineno;
+extern int lineno;  /* FIXME: overloaded with item pointers */
```

Comment:
This could be a union.  Or at least size_t instead of int.  (is that promised to be pointer sized?)